scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -388,11 +388,6 @@ task CreateManifest {
 }
 
 task GenerateSampleListFile {
-  # should not call cache in case of withdrawn samples
-  meta {
-    volatile: true
-  }
-
```

Comment:
do we want to retain this?

Reply:
I think you should. It's entirely dependent on the database state.

Reply:
ok---no longer removed as part of the pr!